lib/rubocop/cop/naming/memoized_instance_variable_name.rb

```diff
@@ -55,6 +55,7 @@ def on_def(node)
           msg = format(
             MSG,
             var: ivar_assign.children.first.to_s,
+            suggested_var: method_name.to_s.sub('?', ''),
```

Comment:
`method_name.to_s.chomp('?')` would work too.

Reply:
I like the `chomp` way, thanks!